lib/generators/sufia/templates/migrations/change_featured_work_generic_file_id_to_generic_work_id.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeFeaturedWorkGenericFileIdToGenericWorkId < ActiveRecord::Migration
+  def change
+    return unless column_exists?(:featured_works, :generic_file_id)
+    rename_column :featured_works, :generic_file_id, :generic_work_id
```

Comment:
@jcoyne the above commit is ok because it works for a first-time installation. When someone is upgrading from 6, they'll need to rename that column because the table already exists. I've put it under the upgrade700 step, but since the engine cart is running this generator as well, we need to skip it since the table and its correct column already exists.

Another option is to make this its own generator step, separate from upgrade700, and then the engine_cart wouldn't call it.